### DISCUSSION TITLE
Components Column width adjustment

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9437] [Help] [answered]
Would it be possible to adjust the components column? With the introduction of the Details side panel, it's is now easily possible to get a tree of screens that partially disappears below the details panel. Thanks in advance!

**Comment by melohagan** [comment, marked as answer]
Hey @ecxs 

There is a horizontal scrollbar along the bottom of the component tree.

**Reply by ecxs** [reply]
Oeps! ... totally missed that ... thanks!
